src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -45,7 +45,7 @@ public class NodeAutoCompleteSearchViewModel : SearchViewModel
         private const string nodeAutocompleteMLEndpoint = "MLNodeAutocomplete";
 
         // Lucene search utility to perform indexing operations just for NodeAutocomplete.
-        private LuceneSearchUtility LuceneUtility
+        private static LuceneSearchUtility LuceneUtility
```

Comment:
could this be moved to a non WPF assembly if this is really a utility - or does it only make sense in a WPF context?
In any case it's confusing to find a class with this name in this .cs file.

Reply:
hmm, I think I got confused by the naming...

the LuceneSearchUtility instances are the actual search implementations for each part of the application. It just gets confusing because the entire thing is singleton with 3 instances inside it, and then whats the difference if one of those instances is static or not on some other object.

In my opinion it's hard to reason about if theres no clear style guide or difference between them in usage since it's all a singleton in the end. I can see a static property on the actual class leading to issues with testing - whereas a non static prop might make it simpler to test with a customized search lib.

Reply:
Not sure why static was added in the first.. Let me remove this